### DISCUSSION TITLE
STR_STATION_LIST_CARGO_FILTER_ALL_AND_NO_RATING update suggestion

### DISCUSSION THREAD

**Discussion by azabost** [OpenTTD/team#502] [Other]
Hey 👋 I'm in the middle of adding some `pl_PL` translations and I noticed a string that could be clarified a little.

The base translation for `STR_STATION_LIST_CARGO_FILTER_ALL_AND_NO_RATING` is "All cargo types and no rating". I downloaded the nightly build to check where that string is displayed and it seems to be this place:

![image](https://github.com/OpenTTD/team/assets/5508894/506c5a25-e03c-4ac0-a971-9a31bbd65235)

Apparently, that string is displayed when the "Select all and no rating" filter is selected, which selects all the the cargo types and the "No cargo rating" option as well.

IMHO it would be easier to understand if the base translation of `STR_STATION_LIST_CARGO_FILTER_ALL_AND_NO_RATING` was "All cargo types **or** no rating" to clearly indicate that it's a [logical disjunction](https://en.wikipedia.org/wiki/Logical_disjunction) (or a [union](https://en.wikipedia.org/wiki/Union_(set_theory))) of both: "all cargo types" and "no rating". With "and" it sounds as if we were displaying the stations of all the cargo types which **also** have no rating (at the same time), which doesn't make sense in this context.

The base translation "All cargo types **and** no rating" is understandable only if you are familiar with the structure of that filter: what items it contains (cargo types) and that one of the options is "no rating", which isn't obvious until you open the popup. However, if you see that string for the first time, you may not understand its meaning immediately.

On the other hand, the "and" conjunction in the "Select all and no rating" string in the popup does make sense because it is displayed in a different context and it expresses a very specific intention: "select X and Y" which means "show me the stations matching **any** of the selected options (X **or** Y)"

What do you think? 🤔

**Comment by glx22** [comment]
Actually it just used to show which options are selected, but the filter will match any of the selected options.
https://github.com/OpenTTD/team/assets/2952192/b84e2a85-fcfc-4c19-be05-cad15c90b3b7

**Reply by azabost** [reply]
Well, yes, I know, I played with it for a while 😊 So don't you think that "All cargo types or no rating" sounds more natural and summarizes that filter better? Please bear in mind you may see that "summary" while the filter popup is closed (so when you don't see the individual options).

Of course, it's just a suggestion on my part, so I will understand if you think differently.
